src/commons/mail-message-renderer.tsx

```diff
@@ -100,7 +100,10 @@ const _TextMessageRenderer: FC<{ body: { content: string; contentType: string }
 	);
 
 	const convertedHTML = useMemo(
-		() => replaceLinkToAnchor(plainTextToHTML(contentToDisplay)),
+		() =>
+			replaceLinkToAnchor(
+				plainTextToHTML(contentToDisplay.replace(/</g, '&lt;').replace(/>/g, '&gt;'))
```

Comment:
Can you consider to move the replace logic inside the replaceLinkToAnchor function? I think it should be handled inside it

Reply:
@nubsthead If we move it in replaceLinkToAnchore function it will create issue by replacing the '<' and '>' character of other html tag. Actually here we need to replace them in plain text so I think we can move it in plainTextToHTML function.